coffea/nanoevents/schemas/physlite.py

```diff
@@ -0,0 +1,50 @@
+import warnings
+from collections import defaultdict
+from coffea.nanoevents.schemas.base import BaseSchema, zip_forms
+
+
+class PHYSLITESchema(BaseSchema):
+
+    mixins = {
+        "Electrons": "xAODParticle",
```

Comment:
Are there no further specializations to electrons/muons/etc. that you'd like to do? It's still a draft so I suppose you may be working on that.

Reply:
Yes, there are going to be more specializations - e.g. the cross references will be different for Electrons/Muons/Jets